tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/SyntaxProvider/SqlServerSyntaxProviderTests.cs

```diff
@@ -186,7 +201,7 @@ public void CreateIndexBuilder_SqlServer_Clustered_CreatesClusteredIndex()
             .Do();
 
         Assert.AreEqual(1, db.Operations.Count);
-        Assert.AreEqual("CREATE CLUSTERED INDEX [IX_A] ON [TheTable] ([A])", db.Operations[0].Sql);
+        Assert.AreEqual("CREATE CLUSTERED INDEX [IX_A] ON [TheTable] ([A] ASC)", db.Operations[0].Sql);
```

Comment:
Not worried about this CodeScene warning, unit test

Reply:
ASC is implicit in the original. This change makes it explict so we can have indexes like ASC, DESC, ASC